src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -128,25 +131,35 @@ const formConfig = {
     formSignature: {
       title: 'Signer information',
       pages: {
-        formSignature: {
-          path: 'form-signature',
+          formSignature: {
+          path: UPLOADS_COMPLETE_PATH,
+          // initialData: mockdata.data,
```

Comment:
Same issue here, I `UPLOADS_COMPLETE_PATH` should be the string added in [this previous PR](https://github.com/department-of-veterans-affairs/vets-website/pull/33388/files#diff-07448f852c91e9cc2cee14e9104a04bc51be6afa5da16c572969be0c1c86c280)